corehq/apps/users/models.py

```diff
@@ -2374,12 +2377,40 @@ def is_member_of(self, domain_qs):
 #
 # Django  models go here
 #
-class Invitation(Document):
+class SQLInvitation(SyncSQLToCouchMixin, models.Model):
+    email = models.CharField(max_length=100, db_index=True)
```

Comment:
I'd say not to worry about django migrations. It'll be easier for everyone to continue using south right now and then once 1.7 is merged we can go through the (hopefully few # of) PRs with migrations and switch them over.